apps/console/src/features/users/components/wizard/bulk-import-user-wizard.tsx

```diff
@@ -0,0 +1,1004 @@
+/**
+ * Copyright (c) 2023, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { IdentityAppsApiException } from "@wso2is/core/exceptions";
+import {
+    AlertLevels,
+    ClaimDialect,
+    ExternalClaim,
+    SCIMResource,
+    SCIMSchemaExtension,
+    TestableComponentInterface
+} from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import {
+    CSVFileStrategy,
+    CSVResult,
+    FilePicker,
+    Heading,
+    LinkButton,
+    PickerResult,
+    PrimaryButton,
+    useWizardAlert
+} from "@wso2is/react-components";
+import { FormValidation } from "@wso2is/validation";
+import Axios from "axios";
+import React, { FunctionComponent, ReactElement, useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+import { useDispatch } from "react-redux";
+import { Dispatch } from "redux";
+import { Dropdown, DropdownItemProps, DropdownProps, Form, Grid, Icon, Modal } from "semantic-ui-react";
+import { v4 as uuidv4 } from "uuid";
+import { getUserStores } from "../../../../extensions/components/users/api";
+import { UsersConstants } from "../../../../extensions/components/users/constants";
+import { userConfig } from "../../../../extensions/configs";
+import { getAllExternalClaims, getDialects, getSCIMResourceTypes } from "../../../claims/api";
+import { UserStoreDetails, UserStoreProperty, getCertificateIllustrations } from "../../../core";
+import { PRIMARY_USERSTORE } from "../../../userstores/constants";
+import { addBulkUsers } from "../../api";
+import {
+    BlockedBulkUserImportAttributes,
+    BulkUserImportStatus,
+    RequiredBulkUserImportAttributes,
+    SpecialMultiValuedComplexAttributes,
+    UserManagementConstants
+} from "../../constants";
+import {
+    BulkResponseSummary,
+    BulkUserImportOperationResponse,
+    BulkUserImportOperationStatus,
+    SCIMBulkEndpointInterface,
+    SCIMBulkOperation,
+    SCIMBulkResponseOperation
+} from "../../models";
+import { BulkImportResponseList } from "../bulk-import-response-list";
+
+/**
+ * Prototypes for the BulkImportUserWizardComponent.
+ */
+interface BulkImportUserInterface extends TestableComponentInterface {
```

Comment:
Extend IdentifiableComponentInterface and remove redundant data-componentid member.

Reply:
Fixed in https://github.com/wso2/identity-apps/pull/6597.